src/components/nodes/boundaryMessageEvent/index.js

```diff
@@ -0,0 +1,31 @@
+import component from './boundaryMessageEvent.vue';
+import boundaryEventConfig from '../boundaryEvent';
+import merge from 'lodash/merge';
+import cloneDeep from 'lodash/cloneDeep';
+
+boundaryEventConfig.inspectorConfig[0].items[0].items.push(
```

Comment:
@chipit24 Might need your input on the best way to get the interrupting checkbox back into the inspector.

Reply:
Calling `push` here mutates the base boundary event config and causes all boundary events to have the interrupting checkbox.

You can look at how it's done in `/src/components/nodes/boundaryTimerEvent/index.js`; it's a little odd with the empty placeholder objects, but it works for now.

I like using `push` though. So instead of exporting the default config object below, assign it to a variable, call `push` on that, and export that. (The `cloneDeep` ensures we're mutating a copy of the config and not the actual base config).